lib/manageiq/providers/workflows/builtin_runner.rb

```diff
@@ -12,31 +12,30 @@ def run_async!(resource, params = {}, secrets = {}, context = {})
 
           method = resource.sub(scheme_prefix, "")
 
+          runner_context = {"method" => method}
```

Comment:
I had to add this so that we know what method was called in the subsequent calls down below.

I'd like to tweak how Floe Task is calling the runner so maybe these are instance methods with the resource as an instance variable so that this is a little cleaner